test/controllers/wikis_controller_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
```

Comment:
Make sure you call your testing directory `spec` instead of `test`, too.

Reply:
see response above.